<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vbc_dialog</class>
 <widget class="QDialog" name="vbc_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Subject Analysis</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_21">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="cal_null_trend">
           <property name="text">
            <string>null trend</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cal_trend">
           <property name="text">
            <string>cal_trend</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="open_subject">
           <property name="text">
            <string>open...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="vbc_threshold">
           <property name="minimum">
            <double>0.500000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.050000000000000</double>
           </property>
           <property name="value">
            <double>0.950000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cal_lesser_tracts">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>FDR</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cal_group_dist">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Group </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="show_null_distribution">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Null</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="vbc_dist_update">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Dist</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="save_vbc_dist">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../icons.qrc">
             <normaloff>:/icons/icons/save.xpm</normaloff>:/icons/icons/save.xpm</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_22">
          <item>
           <widget class="QSplitter" name="splitter_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QCustomPlot" name="null_dist" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
            <widget class="QTableWidget" name="dist_table">
             <column>
              <property name="text">
               <string>voxel</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>pdf(x)</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>cdf(x)</string>
              </property>
             </column>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Subject Data</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QSplitter" name="splitter_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QWidget" name="widget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QGraphicsView" name="vbc_view"/>
               </item>
               <item>
                <widget class="QScrollBar" name="AxiSlider">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QTableWidget" name="subject_list">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <column>
               <property name="text">
                <string>Subject ID</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Value</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>R2</string>
               </property>
              </column>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QCustomPlot" name="vbc_report" native="true"/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>plot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>vbc_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>vbc_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
